sys/cbor/cbor.c

```diff
@@ -522,7 +522,7 @@ size_t cbor_deserialize_float(const cbor_stream_t *stream, size_t offset, float
 
     if (*data == CBOR_FLOAT32) {
         *val = ntohf(*(uint32_t *)(data + 1));
-        return 4;
+        return 5;
```

Comment:
the [doc](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/cbor.h#L335) clearly states that it should be the number of bytes written into `val`. I don't see how `5` can be correct here?